docs/openstack/keystone_adoption.md

```diff
@@ -10,6 +10,29 @@
 
 ## Pre-checks
 
+## Copy over fernet keys
+
+Pull your [openstack configuration using os-diff](pull_openstack_configuration.md) from the old environment.
+
+Create the `keystone` secret, containing fernet keys:
+
+```
+oc apply -f - <<EOF
+apiVersion: v1
+data:
+  CredentialKeys0: $(cat /tmp/collect_tripleo_configs/keystone/etc/keystone/credential-keys/0 | base64 -w 0)
+  CredentialKeys1: $(cat /tmp/collect_tripleo_configs/keystone/etc/keystone/credential-keys/1 | base64 -w 0)
+  FernetKeys0: $(cat /tmp/collect_tripleo_configs/keystone/etc/keystone/fernet-keys/0 | base64 -w 0)
+  FernetKeys1: $(cat /tmp/collect_tripleo_configs/keystone/etc/keystone/fernet-keys/1 | base64 -w 0)
```

Comment:
I wonder if we should utilize `$CONTROLLER_SSH` explicitly instead of reusing what OS Diff pulled into /tmp... That would make it easier to add this into the test suite as it would be decoupled from OS Diff which is not currently there.

Reply:
We discussed yesterday that initially OS Diff will be used after deploying the podified control plane to validate its configuration, but it will not be used before deploying the control plane (to not make the documentation confusing to users). This means we shouldn't rely on OS Diff prior/during deployment of the control plane, we should fetch these keys with $CONTROLLER1_SSH.

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs/openstack/stop_openstack_services.md#variables

Reply:
I changed it to use $CONTROLLER_SSH